recipe.py

```diff
@@ -47,20 +52,35 @@ def run_recipe(self, tombolo_path, output_path, force_imports=None, clear_databa
             `gradle_path`: (Optional) If gradle path is not set in env variables, use this option to pass gradle path.
         """
 
-        gradle = gradle_path + '/bin/' if gradle_path is not None else ''
+        shell = False
+        if platform.system() == 'Windows':
+            self.recipe = self.recipe.replace('\n', '')
+            self.recipe = json.dumps(self.recipe)
+            tombolo_path = tombolo_path.replace(os.sep, ntpath.sep)
+            output_path = output_path.replace(os.sep, ntpath.sep)
+            if gradle_path is not None:
+                gradle_path = gradle_path.replace(os.sep, ntpath.sep)
+            shell = True
+
+        to_save = os.path.join(base_dir, output_path)
+        gradle = os.path.join(gradle_path, 'bin') if gradle_path is not None else ''
         gradle_command = gradle + 'gradle'
-        args = [gradle_command, "runExport", "-Ps",
-                "-Precipe=" + self.recipe, "-Poutput=" + base_dir + '/' + output_path]
+        args = [gradle_command, "runExport", "-Ps", 
+                "-Precipe=" + self.recipe, "-Poutput=" + to_save]
         if force_imports is not None:
             args.append("-Pforce=" + force_imports)
         if clear_database_cache:
             args.append("-Pclear=true")
-        output = sp.Popen(args, stdout=sp.PIPE, cwd=base_dir + '/' + tombolo_path)
+        output = sp.Popen(args, stdout=sp.PIPE, cwd=os.path.join(base_dir, tombolo_path), shell=shell)
 
         for log in iter(output.stdout.readline, b''):
             sys.stdout.write(str(log) + '\n')
         output.stdout.close()
 
+        if platform.system() == 'Windows':
+            sys.exit(0)
```

Comment:
The error didn't gone... Just replaced the one with zero :)

An exception has occurred, use %tb to see the full traceback.

SystemExit: 0

Reply:
Something else that i noticed is that force_imports expects string. Why not boolean?

Reply:
One more thing, what is gradle_path expects? I cant make it work if i hardcode the path...